conf/nutch-default.xml

```diff
@@ -1874,6 +1874,72 @@ visit https://wiki.apache.org/nutch/SimilarityScoringFilter-->
   </description>
 </property>
 
+
+<!-- lib-htmlunit plugin properties; applies to protocol-htmlunit -->
+
+<property>
+  <name>htmlunit.page.load.delay</name>
```

Comment:
Agreed, however it is not a huge deal. I suspect that not many people are using the Selenium plugin properties via the Java API artifacts, hence a trivial name change to share the properties would save us some ~100 lines within nutch-default.xml.